cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/preview/PreviewHttpHandler.java

```diff
@@ -109,4 +114,26 @@ public void getData(HttpRequest request, HttpResponder responder,
     ApplicationId application = namespace.app(previewId);
     responder.sendString(HttpResponseStatus.OK, GSON.toJson(previewManager.getRunner(application).getData(tracerId)));
   }
+
+  @POST
+  @Path("/previews/{preview-id}/tracers/")
+  public void getTracersData(HttpRequest request, HttpResponder responder,
+                             @PathParam("namespace-id") String namespaceId,
+                             @PathParam("preview-id") String previewId) throws Exception {
+    NamespaceId namespace = new NamespaceId(namespaceId);
+    ApplicationId application = namespace.app(previewId);
+    Map<String, List<String>> previewRequest = parseBody(request,
+                                                         new TypeToken<Map<String, List<String>>>() { }.getType());
+    if (previewRequest == null) {
```

Comment:
is the body supposed to be something like:

```
{
  "tracers": ["tracer1", "tracer2", ...] 
}
```

If so, you should also error with a 400 if the right key is not in the map and if the list of tracers is empty